src/font/mod.rs

```diff
@@ -29,6 +29,8 @@ impl<'a> Font<'a> {
             }
             #[cfg(feature = "std")]
             fontdb::Source::SharedFile(_path, data) => data.deref().as_ref(),
+            #[cfg(not(feature = "std"))]
+            _ => { todo!(""); }
```

Comment:
This was needed as a compile-fix for the no_std build.

Reply:
What other enum variant is there?

Reply:
None but 2 of the variants aren't handled in the std case, so I'm defaulting to _ here mostly out of convenience.

Reply:
Looks like my bad, I accidentally used `fontdb` normally as well as one of my dependencies!